changelogs/fragments/docs_argument_specs_version_added.yml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - ansible-doc will now output additional ``version_added`` information from role argument specification information at the entrypoint
+    level, like antsibull-docs has been doing since role support was added.
```

```suggestion
    level, like antsibull-docs has been doing since role support was added (https://github.com/ansible/ansible/pull/80625).
```
I'm not sure whether the `like antsibull-docs has been doing since role support was added` part should be here, but that's for the core team to decide :)

---

Fixed it.